docs/src/testing/basics.md

```diff
@@ -0,0 +1,80 @@
+# Testing Basics
+
+If you're new to Rust, you'll want to review these important tools to help you build tests.
+
+## The `assert!` macro
+
+You can use the `assert!` macro to enforce certain outcomes in your test. This macro throws an error and fails the test if the expression inside evaluates to `false`.
```

Comment:
`assert!()` will not throw an error but invoke `panic!()`.
```suggestion
You can use the `assert!` macro to enforce certain outcomes in your test. This macro invokes `panic!()` and fails the test if the expression inside evaluates to `false`.
```

Reply:
I'm not sure how using `assert` _enforces_ outcomes, rather it _asserts_ (states or declares) that something should be true. Otherwise, the test will panic.